odl/discr/grid.py

```diff
@@ -520,7 +508,7 @@ def __eq__(self, other):
 
     def __hash__(self):
         """Return ``hash(self)``."""
-        # TODO: update with #841
+        # TODO(doable, #841): update (what exactly)
```

Comment:
I know, and I couldn't figure out what the original author of the TODO meant. I just added "doable" because #841 has been merged. The issue number might also be just wrong.

Reply:
Any idea what the original intention of the TODO was? Otherwise I'll delete it.

Reply:
I deleted it now since I can't see anything left to do in this function. The original issue may have been something about hashing of arrays, but I'd say we're good with that for now.